packages/valory/skills/olas_week_abci/prompts.py

```diff
@@ -114,7 +114,7 @@
 https://twitter.com/autonolas/status/1710294338435957248"
 
 Your response should have been:
-"Week 40 in Olas
+"Week 40 '23 in Olas
```

Comment:
24

Reply:
Same question as in https://github.com/valory-xyz/IEKit/pull/117#discussion_r1442853787.